src/System.Device.Gpio/System/Device/Gpio/GpioController.cs

```diff
@@ -323,6 +351,7 @@ private void Dispose(bool disposing)
         public void Dispose()
         {
             Dispose(true);
+            GC.SuppressFinalize(this);
```

Comment:
NIT: I know this is the standard implementation of the Dispose pattern, but the class has no finalizer so this will not have any effect. I would just remove the line.